src/sparseml/yolov5/README.md

```diff
@@ -0,0 +1,126 @@
+# SparseML YOLOv5 Integration
+This directory combines the SparseML recipe-driven approach with the 
+[ultralytics/yolov5](https://github.com/ultralytics/yolov5) repository.
+By integrating the robust training flows in the `yolov5` repository with the SparseML code base, we enable model sparsification techniques on the popular [YOLOv5 architecture](https://github.com/ultralytics/yolov5/issues/280)
```

Comment:
I think the architecture is too tangential to the content here to include directly. In this case the reference was created as an issue, but not meant to be an "issue" rather than a thread of documentation and discussion. Definitely a weird quirk in the Ultralytics repo, but I think it's a choice we have to live with. Also for context, this part of the content is borrowed from the previous YoloV5 documentation written by us.

Reply:
Ok, I agree with the logic.